INTERVIEW_WORKFLOW.md

```
@@ -0,0 +1,227 @@
+This document describes how to conduct technical interviews using Claude's audio capabilities and automatically generate interview rubrics.
+
+## Overview
+
+This workflow allows you to:
+1. Conduct technical interviews using either audio (Claude.ai) or text (Claude Code CLI)
+2. Automatically evaluate the interview and generate a completed rubric
+3. Store interview evaluations for review and comparison
+
+## Two Methods Available
+
+### Method A: Text-Based Interview (Fully Automated)
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)